### ISSUE TITLE
Signup: Consider better visibility for the option to use an existing domain.

### ISSUE THREAD

**Issue by kwight** [Automattic/wp-calypso#22415]
_`/domains` screen_

It's not uncommon for people to come to our service with a domain name they want to use, yet we don't clearly indicate this is supported. Even on large screens, the option to use an existing domain is almost below the fold and not easy to notice.

![screen shot 2018-02-13 at 3 00 26 pm](https://user-images.githubusercontent.com/349751/36178784-e28fd526-10ce-11e8-8afe-c272b3beb726.png)

**Comment by rachelmcr** [comment]
It's perhaps worth noting that the prompt to use an existing domain is more prominent before you start searching, and if you enter an existing domain in the search field.

Before searching:

![screenshot 2018-02-14 12 18 39](https://user-images.githubusercontent.com/8658164/36203956-5645b726-1181-11e8-9547-b89e3d4554db.png)

After searching for an existing domain:

![screenshot 2018-02-14 12 17 57](https://user-images.githubusercontent.com/8658164/36203967-5bd58608-1181-11e8-8289-251045e97eea.png)

**Comment by kwight** [comment]
Oh, interesting – that could mean #22376 would further guarantee it's not surfaced (cc/ @Copons ).

**Comment by Copons** [comment]
> Oh, interesting – that could mean #22376 would further guarantee it's not surfaced

🤔 Very good point.
With my proposed change, by immediately suggesting other domains, we can be pretty sure that the customer would never see their own domain or the notice.
Though, I guess we could move the "Already own a domain" on top (like, after the free `.wordpress.com` suggestion)?

**Comment by gwwar** [comment]
I think there might be an opportunity here to better highlight the mapping option before we start typing in a domain name. At least for me I never noticed it.

**Comment by alisterscott** [comment]
We do now highlight the map option when entering a taken domain:

<img width="1012" alt="screen shot 2018-07-31 at 1 55 44 pm" src="https://user-images.githubusercontent.com/128826/43436830-81182152-94c9-11e8-8f16-31c44322845e.png">

Is that enough to close this issue or does this need some more work?

**Comment by worldomonation** [comment]
Hi! I'm performing triage on older bugs as part of Quality Squad's efforts to cut down on backlog issue counts.

As there has been no recent activity on this issue it is being closed with the label `Closed - Won't Fix`.
If this issue should remain open, please feel free to re-open this bug.
